lib/Client.js

```diff
@@ -1779,10 +1779,11 @@ class Client extends EventEmitter {
     * Modify a guild's widget
     * @arg {String} guildID The ID of the guild
     * @arg {Object} options The widget object to modify (https://discord.com/developers/docs/resources/guild#modify-guild-widget)
```

Comment:
Can you document channelID and enabled? Reassign `channelID` to `channel_id` inside the method ensuring passthrough backwards compatibility